app/views/crime_applications/delete.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <h1 class="govuk-heading-xl"><%=t('.heading', applicant_name: @full_name) %></h1>
+  </div> 
+</div>
+
+<div class: "govuk-body">
```

Comment:
Typo, `class:` -> `class=`

I think all the markup has to be inside grid containers, in this case probably the `<div class="govuk-grid-column-three-quarters">` container?

If not, then we have to open a new `govuk-grid-row`.